### PR TITLE
Fe refacto UI

### PR DESCRIPTION
**Pull request by freddidierRTE**
Nothing in release note

### CODE REVIEW

File: ui/main/src/app/app.module.ts [hunk 1]
```diff
@@ -41,8 +41,10 @@ import {ApplicationLoadingComponent} from './modules/core/application-loading/ap
 import {ReloadRequiredComponent} from './modules/core/reload-required/reload-required.component';
```
Comment (vlo-rte) [inline ui/main/src/app/app.module.ts:41]: Fix copyright

File: ui/main/src/app/business/config/config.server.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import {Observable} from "rxjs";
```
Comment (vlo-rte) [inline ui/main/src/app/business/config/config.server.ts:1]: Missing copyright

File: ui/main/src/app/modules/about/about.component.ts [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 import {Component, OnInit} from '@angular/core';
 import _ from 'lodash-es';
-import {ConfigService} from '@ofServices/config.service';
+import {ConfigService} from 'app/business/config/config.service';
```
Comment (vlo-rte) [inline ui/main/src/app/modules/about/about.component.ts:12]: Fix copyright

File: ui/main/src/app/modules/archives/archives.component.ts [hunk 4]
```diff
@@ -14,7 +14,7 @@ import {ProcessesService} from '@ofServices/processes.service';
 import {Store} from '@ngrx/store';
 import {takeUntil, tap} from 'rxjs/operators';
 import {FormControl, FormGroup} from '@angular/forms';
-import {ConfigService} from '@ofServices/config.service';
+import {ConfigService} from 'app/business/config/config.service';
```
Comment (vlo-rte) [inline ui/main/src/app/modules/archives/archives.component.ts:17]: Fix copyright

File: ui/main/src/app/modules/businessconfigparty/iframedisplay.component.spec.ts [hunk 5]
```diff
@@ -2,7 +2,7 @@ import {IframeDisplayComponent} from './iframedisplay.component';
 import {TestBed} from '@angular/core/testing';
 import {ActivatedRoute, convertToParamMap, Router} from '@angular/router';
 import {of} from 'rxjs';
-import {ConfigService} from '@ofServices/config.service';
+import {ConfigService} from 'app/business/config/config.service';
```
Comment (vlo-rte) [inline ui/main/src/app/modules/businessconfigparty/iframedisplay.component.spec.ts:5]: Fix copyright

File: ui/main/src/app/modules/businessconfigparty/iframedisplay.component.ts [hunk 6]
```diff
@@ -16,7 +16,7 @@ import {Store} from '@ngrx/store';
 import {AppState} from '@ofStore/index';
 import {selectGlobalStyleState} from '@ofSelectors/global-style.selectors';
 import {GlobalStyleService} from '@ofServices/global-style.service';
-import {ConfigService} from '@ofServices/config.service';
+import {ConfigService} from 'app/business/config/config.service';
```
Comment (vlo-rte) [inline ui/main/src/app/modules/businessconfigparty/iframedisplay.component.ts:19]: Fix copyright

File: ui/main/src/app/modules/calendar/calendar.component.ts [hunk 7]
```diff
@@ -23,7 +23,7 @@ import {HourAndMinutes, TimeSpan} from '@ofModel/card.model';
 import {ProcessesService} from '@ofServices/processes.service';
 import {LightCardsStoreService} from '@ofServices/lightcards/lightcards-store.service';
 import {FilterService} from '@ofServices/lightcards/filter.service';
-import {ConfigService} from '@ofServices/config.service';
+import {ConfigService} from 'app/business/config/config.service';
```
Comment (vlo-rte) [inline ui/main/src/app/modules/calendar/calendar.component.ts:26]: Fix copyright

File: ui/main/src/app/modules/card/services/handlebars.service.ts [hunk 8]
```diff
@@ -14,7 +14,7 @@ import {Observable, of} from 'rxjs';
 import {map, tap} from 'rxjs/operators';
 import {ProcessesService} from '@ofServices/processes.service';
 import {DetailContext} from '@ofModel/detail-context.model';
-import {ConfigService} from '@ofServices/config.service';
+import {ConfigService} from 'app/business/config/config.service';
```
Comment (vlo-rte) [inline ui/main/src/app/modules/card/services/handlebars.service.ts:17]: Fix copyright

File: ui/main/src/app/modules/changepassword/changepassword.component.ts [hunk 9]
```diff
@@ -9,7 +9,7 @@
 
 import {Component, OnInit} from '@angular/core';
 import {DomSanitizer, SafeUrl} from '@angular/platform-browser';
-import {ConfigService} from '@ofServices/config.service';
+import {ConfigService} from 'app/business/config/config.service';
```
Comment (vlo-rte) [inline ui/main/src/app/modules/changepassword/changepassword.component.ts:12]: Fix copyright

File: ui/main/src/app/modules/core/application-loading/activityarea-choice-after-login/activityarea-choice-after-login.component.ts [hunk 10]
```diff
@@ -9,7 +9,7 @@
 
 import {Component, TemplateRef, ViewChild} from '@angular/core';
 import {NgbModal, NgbModalRef} from '@ng-bootstrap/ng-bootstrap';
-import {ConfigService} from '@ofServices/config.service';
+import {ConfigService} from 'app/business/config/config.service';
```
Comment (vlo-rte) [inline ui/main/src/app/modules/core/application-loading/activityarea-choice-after-login/activityarea-choice-after-login.component.ts:12]: Fix copyright

File: ui/main/src/app/modules/core/application-loading/application-loading.component.ts [hunk 11]
```diff
@@ -13,7 +13,7 @@ import {Store} from '@ngrx/store';
 import {TranslateService} from '@ngx-translate/core';
 import {AuthenticationService} from '@ofServices/authentication/authentication.service';
 import {CardService} from '@ofServices/card.service';
-import {ConfigService} from '@ofServices/config.service';
+import {ConfigService} from 'app/business/config/config.service';
```
Comment (vlo-rte) [inline ui/main/src/app/modules/core/application-loading/application-loading.component.ts:16]: Fix copyright

File: ui/main/src/app/modules/feed/components/card-list/card-list.component.ts [hunk 12]
```diff
@@ -12,7 +12,7 @@ import {LightCard} from '@ofModel/light-card.model';
 import {Observable} from 'rxjs';
 import {NgbModalRef} from '@ng-bootstrap/ng-bootstrap/modal/modal-ref';
 import {NgbModal} from '@ng-bootstrap/ng-bootstrap';
-import {ConfigService} from '@ofServices/config.service';
+import {ConfigService} from 'app/business/config/config.service';
```
Comment (vlo-rte) [inline ui/main/src/app/modules/feed/components/card-list/card-list.component.ts:15]: Fix copyright

File: ui/main/src/app/modules/feed/components/card-list/filters/filters.component.ts [hunk 13]
```diff
@@ -10,7 +10,7 @@
 import {Component, OnInit} from '@angular/core';
 import {Store} from '@ngrx/store';
 import {AppState} from '@ofStore/index';
-import {ConfigService} from '@ofServices/config.service';
+import {ConfigService} from 'app/business/config/config.service';
```
Comment (vlo-rte) [inline ui/main/src/app/modules/feed/components/card-list/filters/filters.component.ts:13]: Fix copyright

File: ui/main/src/app/modules/feed/components/map/map.component.ts [hunk 14]
```diff
@@ -22,7 +22,7 @@ import WKT from 'ol/format/WKT';
 import Overlay from 'ol/Overlay';
 import {Style, Fill, Stroke, Circle} from 'ol/style';
 import {Attribution, ZoomToExtent, Control, defaults as defaultControls} from 'ol/control';
-import {ConfigService} from '@ofServices/config.service';
+import {ConfigService} from 'app/business/config/config.service';
```
Comment (vlo-rte) [inline ui/main/src/app/modules/feed/components/map/map.component.ts:25]: Fix copyright

File: ui/main/src/app/server/config/angularConfig.server.ts [hunk 15]
```diff
@@ -0,0 +1,35 @@
+import {HttpClient} from '@angular/common/http';
```
Comment (vlo-rte) [inline ui/main/src/app/server/config/angularConfig.server.ts:1]: Missing copyright

File: ui/main/src/tests/mocks/configServer.mock.ts [hunk 16]
```diff
@@ -0,0 +1,24 @@
+import {Observable, ReplaySubject} from 'rxjs';
```
Comment (vlo-rte) [inline ui/main/src/tests/mocks/configServer.mock.ts:1]: Missing copyright

**Review by vlo-rte — CHANGES_REQUESTED** [review]

File: ui/main/src/app/business/config/config.service.ts [hunk 17]
```diff
@@ -54,6 +44,7 @@ export class ConfigService {
     }
 
     public setConfigValue(path: string, value :any) {
+        console.error("Set value: ",value)
```
Comment (quinarygio) [inline ui/main/src/app/business/config/config.service.ts:47]: remove debug log

File: ui/main/src/app/modules/card/services/handlebars.service.ts (hunk 8 above)
Comment (quinarygio) [inline ui/main/src/app/modules/card/services/handlebars.service.ts:17]: Fix copyright

**Review by quinarygio — CHANGES_REQUESTED** [review]

**Review by vlo-rte — APPROVED** [review]

**Pull request merged**